### ISSUE TITLE
 lost items in Toms Storage with drawers

### ISSUE THREAD

**Issue by Isdiria** [MSandro/sky_fabrication3#69]
Copper and Bronze Items working well, but most steel items (Plate, wheels, etc.) are lost when I try to store them in Toms Storage Crafting Terminal.

On the other side sometimes some item duplication happens when crafting Modern Industrialization recipes.

In the classical workbench everything works fine, but it’s no fun to craft all the stuff manual.

**Comment by Isdiria** [comment]
I have discovered the problem, I have drawers connected with Trim and other drawers with an inventory connector at the Drawer access point.
In this setup several items get lost when you add them into the storage ( or duped sometimes while crafting) 
Without attached drawers it seam to work as intended.

**Comment by MSandro** [comment]
Thanks for figuring this out, I know Toms Storage has some issues. I will forward the issue to their github.

**Comment by MSandro** [comment]
@Isdiria please send a screenshot of your setup.

**Comment by Isdiria** [comment]
![2023-02-15](https://user-images.githubusercontent.com/76951613/218998839-3052d467-8a27-4a86-b95a-19c232d88777.png)
![2023-02-15 (29)](https://user-images.githubusercontent.com/76951613/218998845-c3392c3a-b421-401f-b54b-2573b99235df.png)


I detached it from the drawer and everything works fine. For testing I attached it again (with trim). Now the steel stuff seams to work fine, but at least gold plate are removed from the system.
